packages/gatsby-remark-autolink-headers/README.md

```diff
@@ -58,6 +58,7 @@ Note: if you are using `gatsby-remark-prismjs`, make sure that it’s listed aft
 - `maintainCase`: Boolean. Maintains the case for markdown header (optional)
 - `removeAccents`: Boolean. Remove accents from generated headings IDs (optional)
 - `enableCustomId`: Boolean. Enable custom header IDs with `{#id}` (optional)
+- `isIconAfterHeader`: Boolean. Enable the anchor icon to be inline at the end of the header text
```

Comment:
Please append `(optional)` to this option description. Otherwise looks good to me (tried locally)

Reply:
Thanks for the feedback and verifying the PR! Addressed your feedback here 3fe151c